src/message.rs

```diff
@@ -1783,22 +1816,30 @@ pub async fn delete_msgs_ex(
     }
     res?;
 
+    if !deleted_info_msgs.is_empty() {
+        if modified_chat_ids.len() > 1 {
+            bail!("Can only delete from same chat.");
+        }
+        let (info_msg_ids, info_msg_rfc724): (Vec<_>, Vec<_>) =
+            deleted_info_msgs.into_iter().unzip();
+
+        for &msg_id in &info_msg_ids {
+            let msg = Message::load_from_db(context, msg_id).await?;
+            delete_msg_locally(context, &msg).await?;
+        }
+
+        if let Some(chat_id) = modified_chat_ids.iter().next() {
+            send_delete_request(context, chat_id, &info_msg_rfc724).await?;
+        }
```

Comment:
Shouldn't the delete request only be sent if `delete_for_all`?

Reply:
I think webxdc info messages should always be deleted

Reply:
But why delete them for everyone in the group, when the user clicked on "Delete for me" rather than "Delete for everyone"?